libcloud/compute/drivers/gce.py

```diff
@@ -224,14 +243,13 @@ def _merge_response_items(self, list_name, response_list):
         """
         merged_items = {}
         for resp in response_list:
-            if "items" in resp:
-                # example k would be a zone or region name
-                # example v would be { "disks" : [], "otherkey" : "..." }
-                for k, v in resp["items"].items():
-                    if list_name in v:
-                        merged_items.setdefault(k, {}).setdefault(list_name, [])
-                        # Combine the list with the existing list.
-                        merged_items[k][list_name] += v[list_name]
+            # example k would be a zone or region name
+            # example v would be { "disks" : [], "otherkey" : "..." }
+            for k, v in resp["items"].items():
```

Comment:
I see you removed `if "items" in resp`, after merging that PR, I will make the following change (`for k, v in resp.get("items", {}).items()`) to be on the safe side (in case some response for some reason doesn't contains `items` attribute - probably quite unlikely, but just to be on the safe side).

Reply:
EDIT: Sorry, now I see that where this code is used and called it ensures that the item always contains ``items`` attribute so that check is indeed not needed.